docs/pages/preview/dev-process-overview.mdx

```diff
@@ -0,0 +1,170 @@
+
+</Collapsible>
+
+Like any mobile app, the application that is distributed to users is created by compiling (”building”) the Xcode or Android project.
+
+When you initialize a new app with `npx create-expo-app`, you will not see any **android** or **ios** directories. You can generate the native projects by running `npx expo prebuild`, which will initialize the native projects and then apply the project Expo app config (**app.json/app.config.js**) to them ([learn more](https://docs.expo.dev/workflow/prebuild/)).
+
+If you use a cloud-based development workflow, you may never need to run prebuild or install Xcode or Android Studio on your own machine (although you may find this to be useful). This is explained below in the “Local and cloud-based development workflows” section.
+
+<Collapsible summary={`Why aren’t the native projects created by default when you initialize a project with create-expo-app?`}>
+
+The default behavior encourages the use of [Continuous Native Generation](/workflow/continuous-native-generation/) (CNG) to generate the native projects when needed, which can make upgrades and project maintenance significantly easier.
+The following three commands result in more or less the same project:
+
+- `npx create-expo-app MyApp && cd MyApp && npx expo prebuild`
+- `npx create-expo-app --template bare-minimum`
+- `npx react-native init MyApp && cd MyApp && npx install-expo-modules`
+
+</Collapsible>
+
+</Collapsible>
+
+<Collapsible summary={`Continuous Native Generation (CNG)`}>
+
+You can add the native project directories (**android/** and **ios/**) to your **.gitignore** and/or delete the project at any time, then re-generate them from the Expo app config with `npx expo prebuild` whenever they are needed. This is known as Continuous Native Generation. Using CNG can make upgrading to new versions of React Native much easier, it can simplify project maintenance, and it can facilitate setting up complex features such as [App Clips](https://github.com/bndkt/react-native-app-clip), [share extensions](https://github.com/timedtext/expo-config-plugin-ios-share-extension), and [push notifications](https://github.com/OneSignal/onesignal-expo-plugin). [Learn more about CNG](https://docs.expo.dev/workflow/continuous-native-generation/).
+
+
+<Collapsible summary="What if I want to edit the native project configuration in Android Studio or Xcode, rather than generating the projects with prebuild?">
+
+If you prefer to edit the native projects in Android Studio or Xcode, then you can run prebuild once to generate the projects, or initialize your project with `npx create-expo-app --template bare-minimum`, and from then on configure your native projects using those IDEs instead of your from the Expo app config. CNG has proven to be helpful on many teams, but it may not be the best fit for yours, and this is a perfectly legitimate way to use Expo tools.
```

Comment:
If you do this, edit the `ios` directory say, does it mean you can never run `prebuild` again because it would overwrite the changes (and should delete the app config I guess)? It would be helpful to explain what you _lose_ by choosing to manage the native projects yourself.